packages/block-editor/src/components/index.js

```diff
@@ -120,6 +120,7 @@ export {
 export { default as BlockSettingsMenu } from './block-settings-menu';
 export { default as BlockSettingsMenuControls } from './block-settings-menu-controls';
 export { default as BlockTitle } from './block-title';
+export { default as __experimentalUseBlockDisplayTitle } from './block-title/use-block-display-title';
```

Comment:
I think we can just use `BlockTitle` in conjunction with `createInterpolateElement`  and avoid exporting this hook.